src/refactorings/convert-let-to-const/convert-let-to-const.ts

```diff
@@ -0,0 +1,88 @@
+import { Editor, Code, ErrorReason } from "../../editor/editor";
+import { Selection } from "../../editor/selection";
+import * as t from "../../ast";
+import { isLet } from "../../ast";
+import { Binding } from "@babel/traverse";
```

Comment:
@nicoespeon I'm currently importing `@babel/traverse` directly here however was wondering if maybe we'd prefer this behaviour to live in the `ast` module?

Reply:
@nicoespeon I think this is the only change left to make, I looked at the `ast` module but unsure where this should go.

Does it fit in one of the existing files or require a new one?

Reply:
It has to do with `@babel/traverse` so I'd expose it here: https://github.com/nicoespeon/abracadabra/blob/3326ba97cfe1a8049f741148fc12c19ed308729a/src/ast/transformation.ts#L2

Reply:
Done 👍 

Unless you've any other feedback this hopefully we're good to merge :)